jinahub/encoders/audio/VGGISHAudioEncoder/Dockerfile.gpu

```diff
@@ -0,0 +1,12 @@
+FROM nvidia/cuda:11.3.1-cudnn8-runtime
```

Comment:
Make this 11.2, this is max that tensorflow is compatible with
https://www.tensorflow.org/install/source#gpu